Framework/Core/src/DPLWebSocket.cxx

```diff
@@ -286,8 +357,24 @@ void WSDPLHandler::endHeaders()
         break;
       }
     }
+  } else if (false) {
```

Comment:
This part will probably be affected by #7000. You should probably rebase your developments on top of it.